code/game/turfs/simulated/floor.dm

```diff
@@ -141,6 +142,8 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 		return 1
 	if(..())
 		return 1
+	if(pryable == 0)
+		return 1
 	if(intact && istype(C, /obj/item/weapon/crowbar))
```

Comment:
I don't see why it would fail. Also don't do pryable == 0 , if(pryable) is enough.